keps/sig-scheduling/1923-prefer-nominated-node/kep.yaml

```diff
@@ -5,7 +5,7 @@ authors:
 owning-sig: sig-scheduling
 participating-sigs:
   - sig-scheduling
-status: implementable
+status: implemented
```

Comment:
Please revert this change - we mark KEPs as implemented only after the release in which the features goes to stable is actually released (so in this case, only when 1.24 will be out).